internal/app/starter/master_linux.go

```diff
@@ -39,6 +40,13 @@ func createContainer(ctx context.Context, rpcSocket int, containerPid int, e *en
 		fatalChan <- fmt.Errorf("failed to copy unix socket descriptor: %s", err)
 		return
 	}
+
+	apptheusMetric, err := metric.New()
```

Comment:
Is this always enabled?  Shouldn't it be optional?

Reply:
> Is this always enabled? Shouldn't it be optional?

shall we add a new configuration item inside apptainer.conf? or using cli flag to control it? 
Connection failure will be ignored here, but I am also thinking that it is unnessarsary to try for each container creation.